picard/tagger.py

```diff
@@ -489,11 +483,11 @@ def move_files(self, files, target):
         if isinstance(target, (Track, Cluster)):
             for file in files:
                 file.move(target)
-                QtCore.QCoreApplication.processEvents()
+            QtCore.QCoreApplication.processEvents()
         elif isinstance(target, File):
             for file in files:
                 file.move(target.parent)
-                QtCore.QCoreApplication.processEvents()
+            QtCore.QCoreApplication.processEvents()
```

Comment:
Why isn't this called for others `if` branches? I mean since it isn't in the loops, it could be called once after the last `elif`

Reply:
I didn't really knew why it was there, so I thought it was fine. Phillipp made me realize that it is a good idea to keep it where it was.